src/main/scala/com/cognite/spark/datasource/CdpConnector.scala

```diff
@@ -32,16 +32,22 @@ trait CdpConnector {
 
   def getProject(auth: Auth, maxRetries: Int, baseUrl: String): String = {
     val loginStatusUrl = uri"$baseUrl/login/status"
-    val jsonResult = getJson[Data[Login]](auth, loginStatusUrl, maxRetries)
+    val jsonResult = getJson[Data[Login]](auth, loginStatusUrl, maxRetries, None)
     jsonResult.unsafeRunSync().data.project
   }
 
   def baseUrl(project: String, version: String, baseUrl: String): Uri =
     uri"$baseUrl/api/$version/projects/$project"
 
-  def getJson[A: Decoder](auth: Auth, url: Uri, maxRetries: Int): IO[A] = {
+  def getJson[A: Decoder](
+      auth: Auth,
+      url: Uri,
+      maxRetries: Int,
+      applicationId: Option[String]): IO[A] = {
     val result = sttp
       .header("Accept", "application/json")
+      .header("x-cdp-sdk", Constants.SparkDatasourceVersion)
+      .header("x-cdp-app", applicationId.getOrElse("unknown application"))
```

Comment:
In other places we use `"Scala"` but we should define a default value in `Constants` and use that. `"Spark"` is probably a better choice than `"Scala"` since our users can be calling this from PySpark.